### ISSUE TITLE
Crystal SHA256 mismatch

### ISSUE THREAD

**Issue by ylluminate** [Homebrew/homebrew#44372]
Curious if anyone else is seeing an odd SHA256 mismatch when doing a `brew install crystal`?  I'm seeing the following output on attempt to build:
```
==> Downloading https://github.com/crystal/crystal/archive/v0.5.1.tar.gz
==> Downloading from https://codeload.github.com/crystal/autocode-js/tar.gz/v0.5.1
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 6c022ad59a10cec58856dba858a030b745333909108c08bd326c20ac0cb144fb
Actual: 529c00971e066dec246f9e1e1c174720c4f29f2a2185dea674a3d052c5426570
Archive: /Library/Caches/Homebrew/crystal-0.5.1.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by DomT4** [comment]
Confirmed. Have asked upstream about it: https://github.com/crystal/autocode-js/issues/9

**Comment by DomT4** [comment]
Fixed by #44417 being merged.
